Bottom: b164e1cc3fab650a344b2b7a052d1e8e3d6552e4
Top:    7503b9ee0757c55ed3da6b526b9fd44188f4f56f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-27 16:12:33 +0000

Cleanup conditional code now we require a later GTK version


---

diff --git a/src/hid/gtk/gui-dialog.c b/src/hid/gtk/gui-dialog.c
index 1f6b11a..80c1157 100644
--- a/src/hid/gtk/gui-dialog.c
+++ b/src/hid/gtk/gui-dialog.c
@@ -229,14 +229,12 @@ ghid_dialog_close_confirm ()
                           GTK_STOCK_SAVE,            GTK_RESPONSE_YES,
                           NULL);
 
-#if GTK_CHECK_VERSION (2,6,0)
   /* Set the alternative button order (ok, cancel, help) for other systems */
   gtk_dialog_set_alternative_button_order(GTK_DIALOG(dialog),
                                           GTK_RESPONSE_YES,
                                           GTK_RESPONSE_NO,
                                           GTK_RESPONSE_CANCEL,
                                           -1);
-#endif
 
   switch (gtk_dialog_run (GTK_DIALOG (dialog)))
     {
